packages/playwright/tests/iframe.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test } from '../src';
+
+// Required for the skipped cross-origin test below
+// test.use({
+//   launchOptions: {
+//     args: ['--disable-web-security'],
+//   }
+// });
+
+test('DOES NOT snapshot iframe contents when same-origin', async ({ page }) => {
+  await page.goto('/iframe-same-origin');
+  await page
+    .frameLocator('iframe[title="the-iframe"]')
+    .getByRole('button', { name: 'The Button' })
+    .click();
+});
```

Comment:
It'd be good to call out here what clicking the button "would have" snapshotted (I believe it's opening the browser alert box). Just so the intent of this test is clear.